scale/settings.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/python3
+
+""" Shared settings and constant values across multiple scaling scripts"""
+
+# Deprecated constant context values used by Kubernetes
+# NAMESPACES = ['datahub', 'prob140', 'stat28']
+# CLUSTER = 'prod'
+# KUBECTL_CONTEXT = 'gke_data-8_us-central1-a_prod'
+GCLOUD_INSTANCE_GROUP = 'gke-dev-default-pool-dbd2a02e-grp'
+GCE_ZONE = "us-central1-a"
+
+# Legacy scaling settings
+USERS_PER_NODE = 7
+#POD_THRESHOLD = 0.9
+#BUMP_INCREMENT = 2
+
+# New scaling settings #TODO: discuss with BIDS
+CAPACITY_PER_NODE = USERS_PER_NODE  # FIXME: should be determined dynamically
```

Comment:
@ryanlovett did the history100 class with the 8G user pods get deployed already? if so that will need to be taken into account here, else things will be weird.

@tonyyanga why do this by pod count rather than available RAM statistics, which are also avaiable from the kubernetes API and won't have this problem?